WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -271,9 +271,20 @@ public async Task<IActionResult> GetTransactionsAsync([FromQuery, Required]IEnum
 				{
 					await batchingRpc.SendBatchAsync();
 
+					string lastExceptionMessage = null;
 					foreach (var txTask in tasks)
 					{
-						var tx = await txTask;
+						Transaction tx;
+						try
+						{
+							tx = await txTask;
+						}
+						catch (Exception ex)
+						{
+							Logger.LogDebug(ex);
+							lastExceptionMessage = ex.Message;
+							continue;
```

Comment:
What if `Task.WaitAll()` and catch `AggregateException`?

https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.waitall?view=netcore-3.1

Reply:
That's a blocking call. I created a PR with WhenAll instead: https://github.com/zkSNACKs/WalletWasabi/pull/3095